helpers/input_validation.js

```diff
@@ -15,6 +15,18 @@ function input_val (val, regex) {
         return false;
 }
 
+function is_safe_path(path){
+    return !(path.includes("../") || path.includes("..\\"));
```

Comment:
Parent paths are missing: `/..` and `\\..`.

Reply:
I have updated the `is_safe_path` function and added new test cases.